resources/views/translated_models_export.py

```diff
@@ -0,0 +1,55 @@
+from django.http import HttpResponse, HttpResponseBadRequest
+from xlsxwriter.workbook import Workbook
+from modeltranslation.translator import translator, NotRegistered
+from django.apps import apps
+from resources.models.utils import make_translation_excel
+from django.conf import settings
+
+
+def export_translated_models(request):
+    """
+    Returns Excel of translated strings in given models
+
+    Accepts as arguments either *all* which exports models from
+    settings.TRANSLATED_MODELS_EXPORT
+    or *models* as comma separated model name list
+
+    :param request:Django HttpRequest
+    :return:Django HttpResponse
+    """
+
+    response = HttpResponse(content_type='application/vnd.openxmlformats-officedocument.spreadsheetml.sheet')
+    response['Content-Disposition'] = "attachment; filename=test.xlsx"
+
+    if request.GET.get('all') and request.GET.get('models'):
+        return HttpResponseBadRequest("Use only all or models, not both")
+
+    if request.GET.get('all'):
+        models = settings.TRANSLATED_MODELS_EXPORT
```

Comment:
This is not a setting that can be sensibly configured per Respa instance. If you don't want to automatically sniff translated models from the `resources` app, move the setting to `resources/models.py`.